src/include/meta.h

```diff
@@ -46,6 +46,9 @@
 #define WT_METADATA_VERSION "WiredTiger version" /* Version keys */
 #define WT_METADATA_VERSION_STR "WiredTiger version string"
 
+#define WT_MIN_COMPAT_VERSION_MAJOR 3 /* Minimum version we can start on. */
```

Comment:
I think these names are confusing as we also use them to refer to the compatibility version string written to the turtle file by MongoDB. Maybe something like `WT_MIN_UPGRADE_VERSION`?

Reply:
I feel like WT_MIN_DOWNGRADE_VERSION might be the better name here as it the minimum version we can downgrade to? What do you think?